src/contents/all/components/app/App.tsx

```diff
@@ -112,10 +115,12 @@ const App = () => {
     if (request.message === "clicked_browser_action") {
       if (user) {
```

Comment:
@joelrojo So we pass the user from the response of Agent.login from messageListener to authListener, and it stores the session token as it used to, but the user is not being saved in redux so app.tsx doesn't update to remove the login page. Maybe we can go over this in a screen tmo? will work on other tasks in the meantime

Reply:
Yea that sounds good! Can also talk through the popup idea part